### ISSUE TITLE
"save current layout" not working in spyder 5

### ISSUE THREAD

**Issue by raphaelquast** [spyder-ide/spyder#15203]
### What steps will reproduce the problem?

- just click `view` > `window layouts` > `Save current layout`

### Traceback
```python-traceback
Traceback (most recent call last):
  File "D:\USERS\rq\anaconda\envs\DTHydrology\lib\site-packages\spyder\plugins\layout\container.py", line 131, in <lambda>
    triggered=lambda: self.show_save_layout(),
  File "D:\USERS\rq\anaconda\envs\DTHydrology\lib\site-packages\spyder\plugins\layout\container.py", line 307, in show_save_layout
    self.save_current_window_settings('layout_{}/'.format(index),
AttributeError: 'LayoutContainer' object has no attribute 'save_current_window_settings'
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.7.8
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 2012ServerR2

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.2 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  49.6.0.post20210108 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  3.3.3 (OK)
numpy >=1.7                   :  1.19.4 (OK)
pandas >=1.1.1                :  1.1.4 (OK)
scipy >=0.17.0                :  1.5.3 (OK)
sympy >=0.7.3                 :  1.7 (OK)
```

**Comment by ccordoba12** [comment]
Hey @EnderWiggin14, thanks for reporting. This is a duplicate of issue #15129 and it'll be fixed in our next version (5.0.1), to be released shortly.

**Comment by EnderWiggin14** [comment]
I'm confused as to why I was included in the notification email that someone commented on this thread. I didn't post it or previously comment on it.
